modules/nw-sriov-operator-uninstall.adoc

```diff
@@ -66,21 +66,11 @@ $ oc delete crd sriovnetworks.sriovnetwork.openshift.io
 $ oc delete crd sriovoperatorconfigs.sriovnetwork.openshift.io
 ----
 
-. Delete the SR-IOV webhooks:
+. Change the SR-IOV Network Operator configuration custom resource (CR):
 +
 [source,terminal]
 ----
-$ oc delete mutatingwebhookconfigurations network-resources-injector-config
-----
-+
-[source,terminal]
-----
-$ oc delete MutatingWebhookConfiguration sriov-operator-webhook-config
-----
-+
-[source,terminal]
-----
-$ oc delete ValidatingWebhookConfiguration sriov-operator-webhook-config
+$ oc patch sriovoperatorconfig default --type=merge -n openshift-sriov-network-operator   --patch '{ "spec": { "enableOperatorWebhook": false, "enableInjector": false } }'
```

Comment:
This will not work if done after the operator is removed!

This must be done before this step: `Follow the instructions in the "Deleting Operators from a cluster" section to remove the SR-IOV Network Operator from your cluster.`

Reply:
I'm not sure if it's going to work. It would also require test automation/test plan change.